source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.h

```diff
@@ -50,6 +50,11 @@ class JsonTranscoderConfig : public Logger::Loggable<Logger::Id::config>,
           proto_config,
       Api::Api& api);
 
+  // grpc by default doesn't like a frame larger than 4MB. Splitting streamed data
```

Comment:
is 4MB limit in grpc configurable? If yes, does it make sense to make `MaxStreamedPieceSize` configurable?